crates/precompile/Cargo.toml

```diff
@@ -10,20 +10,15 @@ keywords.workspace = true
 categories.workspace = true
 
 [dependencies]
-reth.workspace = true
-
+reth-revm.workspace = true
 p256 = { version = "0.13.2", features = ["ecdsa"] }
 
 [dev-dependencies]
+reth.workspace = true
```

Comment:
do we need this? is this only used for re-exports? if so we should import individual crates

Reply:
this is for the doctest

Reply:
removed it for the doctest